charts/dotnet-core/Chart.yaml

```diff
@@ -5,6 +5,6 @@ name: charts-dotnet-core
 type: application
 dependencies:
 - name: charts-core
-  version: 1.2.1
+  version: 2.0.0
```

Comment:
Update to dotnet-core chart needs to be in a separate PR.
That's the limitation of current pipeline. Since when it runs the charts-core chart is not yet published, dotnet-core chart can't reference it so the publish process will fail.